core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RowLevelFilterExecuterImpl.java

```diff
@@ -94,6 +94,14 @@
    * Applicable for restructure scenarios
    */
   protected boolean[] isMeasurePresentInCurrentBlock;
+  /**
+   * flag to check whether inverted index exists for the filter column
+   */
+  protected boolean isInvertedIndexColumn;
+  /**
+   * flag to check whether the column is included for sorting during data load
+   */
+  protected boolean isSortColumn;
```

Comment:
Please correct me isSortColumn property is for global sort, so filter does not care about whether column was part of global sort or not. Only we need to check whether column data is sorted or not so isSortColumn check and this variable is not required. Please check and handled in all the places if not required